eventbridge/eventbridge.go

```diff
@@ -133,6 +133,9 @@ func Attach(ctx context.Context, source Getter, sink events.Forwarder) <-chan er
 				errChan <- ctx.Err()
 				return
 			default:
+				if source == nil {
```

Comment:
Would it be non-nil in the beginning of Attach but become nil in this select?

Reply:
If there was an error during CreateVM, this will error out because we never set the `Getter` resulting in a panic

Reply:
How about handling the case on the caller side? It is odd to see that we are calling eventbridge.Attach() with nil.